test_psa_target.py

```diff
@@ -163,6 +163,10 @@ def _erase_flash_storage(args, suite):
             "0xFF",
             "0xA1F0000",
             "0XA1FC000",
```

Comment:
I think it will be better clear the complete RAM area, than a specific section for both the targets.

Reply:
Clearing a specific section is a bit faster, but I'm fine with either. The whole non-secure RAM is 0x20040000 to 0x20080000.